hpat/Pandas/Series.py

```diff
@@ -34,38 +41,28 @@
 from hpat.hiframes.pd_series_ext import SeriesType
 
 
-'''
-Pandas Series (https://pandas.pydata.org/pandas-docs/stable/reference/series.html)
-functions and operators definition in HPAT
-Also, it contains Numba internal operators which are required for Series type handling
-
-Implemented operators:
-    at
```

Comment:
Are you sure we don't need "implemented operators" list anymore?

Reply:
As I understood from our last conversation with Sergey Maidanov it was implied that we should remove this list from here. Because we are providing a list of implemented operators below in the file along with the summary lines.